helm/polaris/build.gradle.kts

```diff
@@ -0,0 +1,245 @@
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+plugins {
+  id("base")
+  id("polaris-spotless")
+}
+
+description = "Apache Polaris (incubating) Helm Chart"
+
+val helmTestReportsDir = layout.buildDirectory.dir("reports")
+
+val helmTemplateValidation by
+  tasks.registering(Exec::class) {
+    group = "verification"
+    description = "Run 'helm template' validation with all values files"
+
+    workingDir = projectDir
+
+    val outputFile = helmTestReportsDir.get().file("helm-template/validation.log")
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    mkdir -p ${outputFile.asFile.parent}
+    for f in values.yaml ci/*.yaml; do
+      echo "Validating helm template with ${'$'}f"
+      helm template --debug --namespace polaris-ns --values ${'$'}f .
+    done > ${outputFile.asFile} 2>&1
+  """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "ci/*.yaml", "templates/**/*") }
+    )
+
+    outputs.file(outputFile)
+  }
+
+val helmUnitTest by
+  tasks.registering(Exec::class) {
+    group = "verification"
+    description = "Run Helm unit tests"
+
+    workingDir = projectDir
+
+    val outputFile = helmTestReportsDir.get().file("helm-unit/test.log")
+
+    // Install the plugin if not already installed, then run tests
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    mkdir -p ${outputFile.asFile.parent}
+    helm plugin install https://github.com/helm-unittest/helm-unittest.git 2>/dev/null || true
+    helm unittest . > ${outputFile.asFile} 2>&1
+  """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "templates/**/*", "tests/**/*") }
+    )
+
+    outputs.file(outputFile)
+  }
+
+val chartTestingLint by
+  tasks.registering(Exec::class) {
+    group = "verification"
+    description = "Run chart-testing (lint)"
+
+    workingDir = rootProject.projectDir
+
+    val outputFile = helmTestReportsDir.get().file("ct/lint.log")
+
+    commandLine =
+      listOf(
+        "sh",
+        "-c",
+        """
+    mkdir -p ${outputFile.asFile.parent}
+    ct lint --debug --charts ./helm/polaris > ${outputFile.asFile} 2>&1
+  """
+          .trimIndent(),
+      )
+
+    inputs.files(
+      fileTree(projectDir) { include("Chart.yaml", "values.yaml", "ci/*.yaml", "templates/**/*") }
+    )
+
+    outputs.file(outputFile)
+  }
+
+// Task to build Docker images in minikube's Docker environment
+val buildMinikubeImages by
+  tasks.registering(Exec::class) {
+    group = "build"
+    description = "Build Polaris Docker images in minikube's Docker environment"
+
+    workingDir = rootProject.projectDir
+
+    // Output: file containing SHA digest of built image
+    val outputFile = helmTestReportsDir.get().file("minikube-images.sha256")
```

Comment:
Better relativize `outputFile` to `project.projectDir` (or better `this.workingDir`) here (and in other places) so the `commandLine` doesn't contain any local paths.